News/news.html

```diff
@@ -1,20 +1,49 @@
+<!DOCTYPE html>
 <html>
-    <head>
-        <title>Assignment</title>
-        </head>
-        <body>
-            <table>
-                <tr>
-                    <h2>Latest Stories</h2>
-                </tr>
-                <tr>
-                    <td><img src="images/1.png" height="200" width="250"><figcaption>Published Aug 01, 2022<hr><strong>This is first news</strong></figcaption></td>
-                    <td><img src="images/2.png" height="200" width="250"><figcaption>Published Aug 01, 2022<hr><strong>This is first news</strong></figcaption></td>
-                    <td><img src="images/3.png" height="200" width="250"><figcaption>Published Aug 01, 2022<hr><strong>This is first news</strong></figcaption></td>
-                    <td><img src="images/4.png" height="200" width="250"><figcaption>Published Aug 01, 2022<hr><strong>This is first news</strong></figcaption></td>
-                    <td><img src="images/1.png" height="200" width="250"><figcaption>Published Aug 01, 2022<hr><strong>This is first news</strong></figcaption></td>
-                    <td><img src="images/2.png" height="200" width="250"><figcaption>Published Aug 01, 2022<hr><strong>This is first news</strong></figcaption></td>
-                </tr>
-            </table>
-        </body>
-        </html>
\ No newline at end of file
+<head>
+    <title>Latest News</title>
+    <link rel="stylesheet" href="news.css">
+    </head>
+    <body>
+    <h1>Latest Stories</h1>    
+        <ul>
+        <article>
+        <img src="images/1.png" alt="news1" height="220"width="200" >
+        <figcaption>December 11,2017</figcaption>
+        <p><strong>Masood Barkati, editor of children's mag naunehal, passes away</strong></p>
+        </article>
+
+        <article>
+        <img src="images/2.jpg" alt="news2" height="220"width="200">
+        <figcaption>Updated December 11,2017</figcaption>    
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption  Check this. Figcaption should used along with Figure tab